tests/resources/sample_5.csv

```diff
@@ -1,6 +1,6 @@
-ARID,ESTAB_ARID,UPRN,ADDRESS_TYPE,ESTAB_TYPE,ADDRESS_LEVEL,ABP_CODE,ORGANISATION_NAME,ADDRESS_LINE1,ADDRESS_LINE2,ADDRESS_LINE3,TOWN_NAME,POSTCODE,LATITUDE,LONGITUDE,OA11CD,LSOA11CD,MSOA11CD,LAD18CD,RGN10CD,HTC_WILLINGNESS,HTC_DIGITAL,TREATMENT_CODE
-DDR190314000000195675,DDR190314000000113740,,HH,Household,U,RD06,,Flat 56 Francombe House,Commercial Road,,Windleybury,XX1 0XX,51.4463421,-2.5924477,E00073438,E01014540,E02003043,E06000023,E12000009,1,5,HH_LF3R2E
-DDR190314000000239595,DDR190314000000060908,,HH,Household,U,RD06,,First And Second Floor Flat,39 Cranbrook Road,,Windleybury,XX1 0XX,51.4721166,-2.5970579,E00074083,E01014669,E02003031,E06000023,E12000009,2,4,HH_LF3R3AE
-DDR190314000000320079,DDR190314000000114781,,HH,Household,U,RD06,,Flat 2,60 Coronation Road,,Windleybury,XX1 0XX,51.4451742,-2.5975551,E00074236,E01033361,E02003050,E06000023,E12000009,3,3,HH_LF3R3BE
-DDR190314000000516472,DDR190314000000516472,,HH,Household,U,RD04,,14 Blackhorse Road,,,Windleybury,XX11 0XX,51.4603426,-2.5122237,E00173063,E01015005,E02003117,E06000025,E12000009,4,2,H_LFNR1E
-DDR190314000000522356,DDR190314000000638000,,HH,Household,U,RD06,,18 Marshfield Park Cleeve Wood Road,,,Windleybury,XX11 0XX,51.4958465,-2.513198,E00075275,E01014897,E02003111,E06000025,E12000009,5,1,HH_LF2R3BE
+ARID,ESTAB_ARID,UPRN,ADDRESS_TYPE,ESTAB_TYPE,ADDRESS_LEVEL,ABP_CODE,ORGANISATION_NAME,ADDRESS_LINE1,ADDRESS_LINE2,ADDRESS_LINE3,TOWN_NAME,POSTCODE,LATITUDE,LONGITUDE,OA11CD,LSOA11CD,MSOA11CD,LAD18CD,RGN10CD,HTC_WILLINGNESS,HTC_DIGITAL,TREATMENT_CODE
```

Comment:
Does this file serve any purpose now? Can't see it being used anywhere

Reply:
It doesn't really.  I've removed this file and the `sample_for_print_stories` one as any new sample files for testing can be generated using the sample file generator.